server.php

```diff
@@ -73,7 +110,8 @@ function show_valet_404()
  * Overwrite the HTTP host for Ngrok.
  */
 if (isset($_SERVER['HTTP_X_ORIGINAL_HOST'])) {
-    $_SERVER['HTTP_HOST'] = $_SERVER['HTTP_X_ORIGINAL_HOST'];
+    // Filter host to support xip.io feature
+    $_SERVER['HTTP_HOST'] = support_xip_io($_SERVER['HTTP_X_ORIGINAL_HOST']);
```

Comment:
This can be reverted back to what it previously was; again, we don't actually want to change the HTTP_HOST server variable, it needs to stay as the full `xip.io` URL.